src/index/views.rs

```diff
@@ -9,6 +9,7 @@ CREATE TYPE VectorIndexStat AS (
     idx_sealed BIGINT[],
     idx_growing BIGINT[],
     idx_write BIGINT,
+    idx_size TEXT,
```

Comment:
We can keep it as an integer. We just pass the `SegmentInfo` structure for implementation.

Reply:
What's the benefit?

Reply:
If `SegmentInfo` is exported in the future, it will be possible for users to get detailed information by reading JSON.

Reply:
Do you have any documents about querying int column as JSON?

Reply:
I mean `stat` RPC might be exported as a Postgresql function returning JSON. So it will be possible to take advantages of Postgresql JSON functions.

Reply:
Got it. Fixed.